src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -91,8 +92,8 @@ def show_read_all_button?
     fetch_notifications.count > Notification::MAX_PER_PAGE
   end
 
-  def send_notifications_information_rabbitmq(number_of_records)
+  def send_notifications_information_rabbitmq(number_of_marked_as_read)
     RabbitmqBus.send_to_bus('metrics',
-                            "notification.delivered,web=true,rss=false value=#{number_of_records}")
+                            "notification.marked_as_read,web=false,rss=true value=#{number_of_marked_as_read}")
```

Comment:
I would suggest we switch to a `action` tag instead of hard coding this into the series name. Makes it easier to mix this into one query. There is also *no* way to mark an RSS notification as read so we don't need to track this at all. So like:

```suggestion
                            "notification,action=read value=#{number_of_marked_as_read}")
```

Reply:
Same way you could track the other number (number_of_marked_as_unread)...